README.md

```diff
@@ -17,7 +17,7 @@
 
 ## Application Page url
 
-https://phelian23.github.io/to-do-list/
+Here is a link to the [live](https://phelian23.github.io/to-do-list/) demo
 
```

Comment:
[Optional]

- [ ] You should update the DOM elements instead of refreshing the page after every operation.

Reply:
resolved this

Reply:
- [ ] Fix live demo link